lib/Installation/ConfirmInstallation.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Confirm Installation Page.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::ConfirmInstallation;
+use Installation::Popups::AcceptPopup;
+use Installation::Popups::OKPopup;
+use strict;
+use warnings;
+use testapi qw(save_screenshot record_info);
+use YuiRestClient;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+    return $self->init();
+}
+
+sub init {
+    my $self = shift;
+    $self->{popup_OK} = Installation::Popups::OKPopup->new({app => YuiRestClient::get_app()});
+    $self->{popup_accept} = Installation::Popups::AcceptPopup->new({app => YuiRestClient::get_app()});
+    $self->{rct_license} = $self->{app}->richtext({id => "lic"});
+    $self->{lbl_license} = $self->{app}->label({debug_label => "Confirm Package License: java-1_..."});
+    return $self;
+}
+
+sub accept_license_if_shown {
+    my ($self) = @_;
+    if ($self->{rct_license}->exist({timeout => 10})) {
```

Comment:
wait 10 seconds 99.999 of the time that a job in product or maintenance run, except for that single one time that we got this strange java popup 1 per year? I don't think it is a good idea...

I would prefer to click on confirm installation if it is found and if it is not, then trigger that logic to detect if there is java popup, and if there is confirm installation afterward, wdyt?

Reply:
Done